examples/with-google-tag-manager/_pages/_app.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect } from 'react'
+import Script from 'next/script'
```

Comment:
What is this exactly?

Reply:
E.g. why is this `_pages`? Is it not being used?

Reply:
> Do we think we wait to update the next/script docs to link back to these once it's further in stability? 

Yeah I think once we start landing a couple more third-parties, we can update the `next/script` docs in the coming weeks to point to this :)

> What is this exactly? Why is this _pages? Is it not being used?

Yeah apologies. This was my initial attempt to update the recommended example and my original plan was to use `_pages` to demonstrate a Pages Router example alongside `app`. I removed the example in this PR altogether and [57364](https://github.com/vercel/next.js/pull/57364) includes an updated example (in App Router format).